### DISCUSSION TITLE
Beta feedback for Live Activities

### DISCUSSION THREAD

**Discussion by jrytio** [gdombiak/OctoPod#636] [General]
Hey @gdombiak I've installed the latest beta as well as OP plug-in and have Live Activities and Lock Screen widgets configured and working. 

I'm curious what the 'live' expectation is on these?  I seem to be getting updates on a polling (or after I open the app).  Not as real time as the name Live Activities made me think.  It doesn't seem a lot faster than the previous generation Home Screen widget was to update. 

Is this expected or is something not working for me?  

iPhone 13 Pro 16.3.1
OctoPod OP Plug-in 0.3.15
OctoPod 3.2.6(1) from March 19 in TestFlight 

Example:

The printer was at 26.4% when this screenshot was taken. 

![image](https://user-images.githubusercontent.com/12903320/226772136-349f28a9-cb7c-4c53-bf03-ad49de055314.png)

**Comment by gdombiak** [comment]
Hey @tideline3d,

Thanks for your feedback. I'm sure that your question will become a common question from many other people. Apple has so many different but yet similar features that may end up confusing people. I just created this Wiki page [Widgets, Live Activities and Notifications](https://github.com/gdombiak/OctoPod/wiki/Widgets,-Live-Activities-and-Notifications) with the hope to clarify things.

In short, things are working as expected. Widgets in the Lock Screen (the 2 things at the top of your screenshot) are updated every 5 minutes when printing. They do not require the OctoPod plugin since they have like a background timer when it's time to be updated. The Live Activity at the bottom of your screenshot gets updated via the OctoPod plugin and that is updated way more frequently (based on progress made by your print).

Let me know if things are not working as I described.

Thanks,
Gaston
